examples/with-typescript-pages/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue';
+
+declare global {
+  namespace JSX {
```

Comment:
Shouldn't we move this typings into `vue-app` package?

Reply:
I agree with @pi0, is it necessary to have this file in the example @chanlito?